CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # CHANGELOG
+## 0.0.37
+- [spigot] Improved namedSpaceKey's names for EnchantGlow, now won't collide with other plugins.
```

Comment:
- [spigot] Change EnchantGlow's key and name so it wouldn't collide with other plugins